src/BootConfiguration.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace LotGD\Core;
+
+use Composer\Package\PackageInterface;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Yaml\Yaml;
+
+use LotGD\Core\ComposerManager;
+
+/**
+ * BootConfiguration
+ * @author sauterb
+ */
+class BootConfiguration
+{
+    /** @var ComposerManager */
+    private $composerManager;
+    /** @var PackageInterface */
+    private $package;
+    /** @var string */
+    private $rootNamespace;
+    /** @var array */
+    private $rawConfig;
+    private $models;
+    private $daenerysCommands;
+    
+    public function __construct(ComposerManager $composerManager, PackageInterface $package)
+    {
+        $this->composerManager = $composerManager;
+        $this->package = $package;
+        
+        $installationManager = $composerManager->getComposer()->getInstallationManager();
+        $confFile = $installationManager->getInstallPath($package)  . "/lotgd.yml";
```

Comment:
I would also check here for this file to exist and throw an exception if it doesnt.